applications/nrf_desktop/src/events/usb_event.h

```diff
@@ -26,40 +20,59 @@
 extern "C" {
 #endif
 
+/**
+ * @brief USB Event
+ * @defgroup nrf_desktop_usb_event USB Event
+ *
+ * File defines a set of events used to transmit the information about USB
+ * state between the application modules.
+ *
+ * @{
+ */
 
 /** @brief Peer states. */
 enum usb_state {
-	USB_STATE_DISCONNECTED, /**< Cable is not attached. */
-	USB_STATE_POWERED,      /**< Cable attached but no communication. */
-	USB_STATE_ACTIVE,       /**< Cable attached and data is exchanged. */
-	USB_STATE_SUSPENDED,    /**< Cable attached but port is suspended. */
+	/** Cable is not attached. */
```

Comment:
Just an opinion - the inline version is easier to read.

Reply:
I also prefer inline, and for the same reason. It's just @divipillai pointed to me the doc style which requires enums to have documentation placed above the values.

Reply:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#enums - Followed this guideline.